internal/cmd/plan.go

```diff
@@ -463,10 +463,20 @@ func promptPlanSelection(plans []turso.Plan, current string) (string, error) {
 		planNames = append(planNames, plan.Name)
 	}
 
+	settings, err := settings.ReadSettings()
+	if err != nil {
+		return "", err
+	}
+
+	var org string
+	if org = settings.Organization(); org == "" {
```

Comment:
```suggestion
	if org := settings.Organization(); org == "" {
```
you can ditch the previous declaration and packing var declaration and assignment in the same line here.

Reply:
I don't think I can, because org is reassigned within the branch. 

```
# github.com/tursodatabase/turso-cli/internal/cmd
internal/cmd/plan.go:478:80: undefined: org
```

Reply:
you are right. I think the following would be a bit more readable then, but ⛏️⛏️ ⛏️  
```suggestion
        org := settings.Organization()
	if org == "" {
```